dotCMS/src/main/resources/es-sitesearch-mapping.json

```diff
@@ -5,64 +5,59 @@
           "type": "multi_field",
           "fields": {
             "untouched": {
-              "type": "string",
-              "index": "not_analyzed",
+              "type": "keyword",
               "ignore_above": 8191,
               "doc_values" : true
             },
             "content": {
-              "type": "string",
+              "type": "text",
               "analyzer": "standard_content"
             },
             "ngram": {
               "search_analyzer": "standard_content",
-              "index_analyzer": "partial_content",
-              "type": "string"
+              "analyzer": "partial_content",
+              "type": "text"
             }
           }
         },
         "host": {
-          "type": "string",
-          "index": "not_analyzed",
+          "type": "keyword",
           "doc_values" : true
         },
         "contentLength": {
           "type": "long"
         },
         "uri": {
-          "type": "string",
-          "index": "not_analyzed",
+          "type": "keyword",
           "doc_values" : true
         },
         "url": {
-          "type": "string",
-          "index": "not_analyzed",
+          "type": "keyword",
           "doc_values" : true
         },
         "mimeType": {
-          "type": "string",
-          "index": "not_analyzed",
+          "type": "keyword",
           "doc_values" : true
         },
         "title": {
-          "type": "string"
+          "type": "text"
         },
         "description": {
-          "type": "string"
+          "type": "text"
         },
         "modified": {
           "type": "date",
           "doc_values" : true
         },
         "keywords": {
-          "type": "string",
-	 	  "analyzer": "comma_analyzer"	
+          "type": "keyword",
```

Comment:
"keyword" or "text" here?

Reply:
I suggest **keyword** datatype if it is really intended for ES aggregations : https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html . Do you know what its purpose is in ESSiteSearch? I haven't tested this functionality yet, so I don't know how it works. To make the decision we have to consider these aspects:

* Will "keywords" field be used for aggregations? 
* Is it intended for full text searching?
* Do we want to keep analyzers for this field? 

Answering these questions will help us to decide what datatype use